datetime_formatter/tests/test_best_matcher.py

```diff
@@ -50,7 +50,9 @@ def test_user(self):
                 self.rl.with_context(lang=False).best_match().code, lang.code
             )
             # Lang not found in context
-            self.assertEqual(self.rl.with_context(dict()).best_match().code, lang.code)
+            self.assertEqual(
+                self.rl.with_context(**dict()).best_match().code, lang.code
```

Comment:
`pylint` seems to be mistaken here, this does not seem like a safe change.

The old call in version 14 used an empty dict, which means `with_context` empties the context, removing any `lang` key if it was present.

However this new version unpacks an empty dictionary, which will not override anything. In particular in this test, if `lang` is in the context, this will _not_ clear it, thus making it uncertain whether this test is actually testing the correct functionality.
To placate `pylint`, consider changing it to perhaps use `{}` instead of `dict()`. If that doesn't work, you may be able to [explicitly ignore it](https://stackoverflow.com/a/29562085).

Reply:
{} did the trick. 
Now pylint has some issues with readme file, though README.rst, readme folder with CONTRIBUTORS.rst, DESCRIPTION.rst, USAGE.rst is present in the module..